codalab/server/bundle_manager.py

```diff
@@ -692,6 +693,25 @@ def _get_staged_bundles_to_run(self):
                     bundle,
                     {'state': State.FAILED, 'metadata': {'failure_message': failure_message}},
                 )
+            elif bundle.metadata.request_queue:
+                # For those bundles that were requested to run on a private worker, if the private worker does not exist
+                # temporarily, we dispatch those bundles until the private worker connected to CodaLab server.
+                request_queue = bundle.metadata.request_queue
+                if request_queue:
+                    tagm = re.match('tag=(.+)', request_queue)
+                if not ([worker for worker in workers.workers() if worker['tag'] == tagm.group(1)]):
+                    # TODO: update bundles with batch operation: implement function batch_update_bundles()
```

Comment:
Not sure you can do that if the message is different for each bundle

Reply:
Found some information online related to this: [link 1](https://stackoverflow.com/questions/54365873/sqlalchemy-update-multiple-rows-in-one-transaction), [link 2](https://docs.sqlalchemy.org/en/13/_modules/examples/performance/bulk_updates.html), [link 3](https://www.tutorialspoint.com/sqlalchemy/sqlalchemy_orm_updating_objects.htm)